cryosparc/job.py

```diff
@@ -321,7 +321,7 @@ def connect(self, target_input: str, source_job_uid: str, source_output: str, re
             self.refresh()
         return result
 
-    def disconnect(self, target_input: str, connection_idx: Optional[int] = None, refresh: bool = True):
```

Comment:
what is the purpose of these `*` added to the signatures?

Reply:
Adding this `*` means that all arguments that follow _must_ as keyword args (e.g., `disconnect(..., refresh=True)`, `disconnect(..., True)` is invalid.

For some cases it's to make the type checker happy.  For example, I override `connect` in `ExternalJob` to include additional arguments. But because the positional args don't match the original implementation, Pyright complains. 

For other cases, such as here, it's for consistency with all the required type checker changes. It's also clearer when you're using it: A random `True` argument in a function call is harder to read than an explicit `refresh=True`.